File: scratch_nav_check.js
const fs = require('fs');
const files = fs.readdirSync('.').filter(f => f.endsWith('.html'));
files.forEach(f => {
  const content = fs.readFileSync(f, 'utf8');
  const navIdx = content.indexOf('id="navMenu"');
  if (navIdx !== -1) {
    const endIdx = content.indexOf('</ul>', navIdx);
    const navContent = content.substring(navIdx, endIdx);
    const linksCount = (navContent.match(/<li/g) || []).length;
    console.log(`${f}: ${linksCount} links`);
  } else {
    console.log(`${f}: NO NAVMENU`);
  }
});

File: scratch_nav_check.py
import os

for f in os.listdir('.'):
    if f.endswith('.html'):
        with open(f, 'r', encoding='utf-8') as file:
            content = file.read()
            navIdx = content.find('id="navMenu"')
            if navIdx != -1:
                endIdx = content.find('</ul>', navIdx)
                navContent = content[navIdx:endIdx]
                linksCount = navContent.count('<li')
                print(f"{f}: {linksCount} links")
            else:
                print(f"{f}: NO NAVMENU")
